opensearch/src/main/java/org/opensearch/sql/opensearch/setting/OpenSearchSettings.java

```diff
@@ -98,6 +102,25 @@ public class OpenSearchSettings extends Settings {
       Setting.Property.NodeScope,
       Setting.Property.Dynamic);
 
+  // we are keeping this to not break upgrades if the config is already present.
+  // This will be completely removed in 3.0.
+  public static final Setting<InputStream> DATASOURCE_CONFIG = SecureSetting.secureFile(
```

Comment:
why move to opensearch module? would it be better in datasource module?

Reply:
Wanted to make OpenSearchSettings class a single interface for all settings. 
* This class is an extension for Setting class in commons module and available for all modules.
* This class has also got the implementation of consuming latest updates.